src/core/GodotGlobal.cpp

```diff
@@ -88,6 +91,16 @@ void Godot::gdnative_init(godot_gdnative_init_options *options) {
 					extension = extension->next;
 				}
 			} break;
+			case GDNATIVE_CORE: {
+			        godot::nativescript_api = (const godot_gdnative_ext_nativescript_api_struct *)godot::api->extensions[i];
```

Comment:
This line should be only in `case GDNATIVE_EXT_NATIVESCRIPT:`.

Reply:
I've updated it, is it correct now?

Reply:
Hmm, I do not think core api should be in the same loop at all, it is not extension but part of core api:

```c++
void Godot::gdnative_init(godot_gdnative_init_options *options) {
	godot::api = options->api_struct;
	godot::gdnlib = options->gd_native_library;

	const godot_gdnative_api_struct *core_extension = godot::api->next;

	while (core_extension) {
		if (core_extension->version.major == 1 && core_extension->version.minor == 1) {
			godot::core_1_1_api = (const godot_gdnative_core_1_1_api_struct *) core_extension;
		}
		core_extension = core_extension->next;
	}

	// now find our extensions
	for (int i = 0; i < godot::api->num_extensions; i++) {
		switch (godot::api->extensions[i]->type) {
			case GDNATIVE_EXT_NATIVESCRIPT: {
				godot::nativescript_api = (const godot_gdnative_ext_nativescript_api_struct *)godot::api->extensions[i];

				const godot_gdnative_api_struct *extension = godot::nativescript_api->next;

				while (extension) {
					if (extension->version.major == 1 && extension->version.minor == 1) {
						godot::nativescript_1_1_api = (const godot_gdnative_ext_nativescript_1_1_api_struct *)extension;
					}

					extension = extension->next;
				}
			} break;
			default: break;
		}
	}
}
```

Reply:
Should be fixed now.